Makefile

```
@@ -52,7 +52,7 @@ GIT_DEPS := $(O)/DEPS/toml11
 all: $(GIT_DEPS) $(PROJECT)
 
 $(PROJECT): $(OBJS)
-	$(CXX) $(CXXFLAGS) $^ $(LIBS) -o $@
+	$(CXX) $(CXXFLAGS) $^ $(LIBS) -o $@ $(LDFLAGS)
```

```suggestion
	$(CXX) $(CXXFLAGS) $^ $(LIBS) $(LDFLAGS) -o $@
```

Just a style comment, but I prefer this.

---

Done.